editor/src/uuiui/inputs/number-input.tsx

```diff
@@ -760,19 +760,35 @@ export const NumberInput = betterReactMemo<NumberInputProps>(
           ) : null}
         </div>
         {DEPRECATED_labelBelow == null && controlStatus != 'off' ? null : (
-          <div
-            onMouseDown={onLabelMouseDown}
-            style={{
-              cursor: CSSCursor.ResizeEW,
-              fontSize: 9,
-              textAlign: 'center',
-              display: 'block',
-              color: controlStyles.secondaryColor,
-              paddingTop: 2,
-            }}
-          >
-            {DEPRECATED_labelBelow != null ? DEPRECATED_labelBelow : null}
-          </div>
+          <React.Fragment>
+            {isFauxcused ? (
+              <div
+                css={{
```

Comment:
Any reason why you used `css` here rather than `style`?

Reply:
No, left over from debugging (via label / hover). Fixed.